scaleway/data_source_rdb_acl_test.go

```diff
@@ -44,34 +39,41 @@ func TestAccScalewayDataSourceRDBAcl_Basic(t *testing.T) {
 			},
 			{
 				Config: fmt.Sprintf(`
-					resource "scaleway_rdb_instance" "main" {
+					resource "scaleway_rdb_instance" "rdbAcl" {
 						name = "%s"
 						node_type = "db-dev-s"
 						engine = "PostgreSQL-12"
 						is_ha_cluster = false
 					}
 
 					resource "scaleway_rdb_acl" "main" {
-						instance_id = scaleway_rdb_instance.main.id
+						instance_id = scaleway_rdb_instance.rdbAcl.id
 						acl_rules {
 							ip = "1.2.3.4/32"
 							description = "foo"
 						}
-
-						acl_rules {
-							ip = "4.5.6.7/32"
-							description = "bar"
-						}
 					}
 					data "scaleway_rdb_acl" "maindata" {
-						instance_id = scaleway_rdb_instance.main.id
+						instance_id = scaleway_rdb_instance.rdbAcl.id
 
 					}`, instanceName),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr("scaleway_rdb_acl.main", "acl_rules.0.ip", "1.2.3.4/32"),
 					resource.TestCheckResourceAttr("data.scaleway_rdb_acl.maindata", "acl_rules.0.ip", "1.2.3.4/32"),
```

Comment:
could you also keep the second acl_rules block and check if the 4.5.6.7 is also present?